boards/sim/sim/sim/configs/adb/defconfig

```diff
@@ -0,0 +1,112 @@
+#
+# This file is autogenerated: PLEASE DO NOT EDIT IT.
+#
+# You can use "make menuconfig" to make any modifications to the installed .config file.
+# You can then do "make savedefconfig" to generate a new defconfig file that includes your
+# modifications.
+#
+# CONFIG_NSH_CMDOPT_HEXDUMP is not set
+# CONFIG_SIG_SIGPIPE_ACTION is not set
+# CONFIG_SIG_SIGSTOP_ACTION is not set
+CONFIG_ADBD_AUTHENTICATION=y
+CONFIG_ADBD_AUTH_PUBKEY=y
+CONFIG_ADBD_BOARD_INIT=y
+CONFIG_ADBD_DEVICE_ID="serialno"
+CONFIG_ADBD_FILE_SERVICE=y
+CONFIG_ADBD_LOGCAT_SERVICE=y
+CONFIG_ADBD_NET_INIT=y
+CONFIG_ADBD_SHELL_SERVICE=y
+CONFIG_ADBD_TCP_SERVER=y
+CONFIG_ARCH="sim"
+CONFIG_ARCH_BOARD="sim"
+CONFIG_ARCH_BOARD_SIM=y
+CONFIG_ARCH_CHIP="sim"
+CONFIG_ARCH_SIM=y
+CONFIG_BOARDCTL_APP_SYMTAB=y
+CONFIG_BOARDCTL_POWEROFF=y
+CONFIG_BOARD_LOOPSPERMSEC=0
+CONFIG_BOOT_RUNFROMEXTSRAM=y
+CONFIG_BUILTIN=y
+CONFIG_DEBUG_ERROR=y
+CONFIG_DEBUG_FEATURES=y
+CONFIG_DEBUG_INFO=y
+CONFIG_DEBUG_NET=y
+CONFIG_DEBUG_NET_ERROR=y
+CONFIG_DEBUG_NET_WARN=y
+CONFIG_DEBUG_SYMBOLS=y
+CONFIG_DEBUG_WARN=y
+CONFIG_DEV_FIFO_SIZE=64
+CONFIG_DEV_LOOP=y
+CONFIG_DEV_PIPE_MAXSIZE=64
+CONFIG_DEV_PIPE_SIZE=64
+CONFIG_DEV_URANDOM=y
+CONFIG_DEV_ZERO=y
+CONFIG_EVENT_FD_NPOLLWAITERS=4
+CONFIG_EXAMPLES_HELLO=y
+CONFIG_EXAMPLES_HELLO_STACKSIZE=7168
+CONFIG_EXAMPLES_WEBSERVER=y
+CONFIG_EXAMPLES_WEBSERVER_DRIPADDR=0x0a000101
+CONFIG_EXAMPLES_WEBSERVER_IPADDR=0x0a000102
+CONFIG_FAT_LCNAMES=y
+CONFIG_FAT_LFN=y
+CONFIG_FSUTILS_PASSWD=y
+CONFIG_FSUTILS_PASSWD_READONLY=y
+CONFIG_FS_BINFS=y
+CONFIG_FS_FAT=y
+CONFIG_FS_PROCFS=y
+CONFIG_FS_ROMFS=y
+CONFIG_FS_TMPFS=y
+CONFIG_IDLETHREAD_STACKSIZE=4096
+CONFIG_LIBC_EXECFUNCS=y
+CONFIG_LIBUV=y
+CONFIG_LIBUV_FS_POLL=y
+CONFIG_LIBUV_NPOLLWAITERS=16
+CONFIG_LIBUV_PIPE=y
+CONFIG_LIBUV_TCP=y
+CONFIG_LIBUV_UNIT_TESTS=y
```

Comment:
remove the unit tests?

Reply:
I would prefer to keep it in order to build this at least in one target so I know when libuv breaks again at compile time.

Reply:
how about we add new project for uv